tests-fuzz/src/translator/postgres/alter_expr.rs

```diff
@@ -112,15 +118,40 @@ impl AlterTableExprTranslator {
             .collect::<Vec<_>>()
             .join(" ")
     }
+
+    fn format_set_table_options(name: impl Display, options: &[AlterTableOption]) -> String {
+        format!(
+            "ALTER TABLE {name} SET {};",
+            options
+                .iter()
+                .map(|option| option.to_string())
+                .collect::<Vec<_>>()
+                .join(", ")
+        )
+    }
+
+    fn format_unset_table_options(name: impl Display, keys: &[String]) -> String {
+        format!(
+            "ALTER TABLE {name} UNSET {};",
+            keys.iter()
+                .map(|key| format!("'{}'", key))
+                .collect::<Vec<_>>()
+                .join(", ")
+        )
+    }
```

Comment:
Is it possible to share some codes between `tests-fuzz/src/translator/mysql/alter_expr.rs` and `tests-fuzz/src/translator/postgres/alter_expr.rs`?